internal/repos/store.go

```diff
@@ -570,6 +570,22 @@ func (s *Store) UpsertRepos(ctx context.Context, repos ...*types.Repo) (err erro
 	return nil
 }
 
+// EnqueueSingleSyncJob enqueues a single sync job for the given external
+// service if it is not already queued or processing.
+func (s *Store) EnqueueSingleSyncJob(ctx context.Context, id int64) (err error) {
```

Comment:
It's not really enqueuing if there can only be one item in the queue?
```suggestion
func (s *Store) CreateExternalServiceSyncJob(ctx context.Context, id int64) (err error) {
```

Reply:
I wanted to use `Enqueue` to make it clear that the job isn't starting now, just being queued up to be started later.